scripts/loadEmptyKeys.js

```diff
@@ -0,0 +1,137 @@
+/*!
+Copyright 2018 Raising the Floor US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script generates unique keys and loads them into CouchDB.
+//
+// To create 20 empty keys, the script needs to be called like this:
+//   * COUCHDB_URL=http://localhost:8058/gpii NUM_OF_KEYS=20 node scripts/loadEmptyKeys.js
+//
+// The options to this script must be passed as environment variables, and they are:
+//   * COUCHDB_URL [String] [required]: This needs to be in the form "http://host/dbname", but in case
+//   you need to provide the credentials to CouchDB, you need to use the form
+//   "http://user:pass@host:port/dbname".
+//   * NUM_OF_KEYS [Integer] [required)]: The amount of new empty preferences sets you want to create.
+//   * OUTPUT_FILENAME [String] [optional]: The file to save the list of keys created. By default,
+//   it will generate a file called "generated-keys-<TIMESTAMP>.txt", just provide a
+//   different value if you want it to be a different file.
+//   * HIDE_PROGRESS [Integer] [optional]: Whether you want to hide the keys as they are created. Default is
+//   0 (show progress), set it to 1 if you want it to be the other way round.
+//
+// See this JIRA for more information: https://issues.gpii.net/browse/GPII-3381
+//
+"use strict";
+
+var process = require("process"),
+    fs = require("fs"),
+    fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    uuid = uuid || require("node-uuid");
+
+require("./shared/prefsSetsDbUtils.js");
+require("gpii-pouchdb");
+
+fluid.defaults("gpii.uuidLoader", {
+    gradeNames: ["gpii.pouch"],
+    outputFileName: process.env.OUTPUT_FILENAME || "generated-keys-" + new Date().toISOString() + ".txt",
+    totalNumOfKeys: parseInt(process.env.NUM_OF_KEYS),
+    hideProgress: process.env.HIDE_PROGRESS,
+    count: 1,
```

Comment:
`count` is not intended to be configurable but more for internal tracking, so it might make more sense to have it as a member option.